### PR TITLE
AQ-51_factory_boy_faker

### PR DESCRIPTION
**Pull request by martynawitkowska**
# Description

Installed Factory Boy / Faker packages. Created a simple factory for generating test users. Added custom management command for generating test data.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The related memme has been added
fake-data-fake

### CODE REVIEW

**Comment by martynawitkowska** [comment]
I was thinking if creating a custom app for data generator makes sense. Right now the generator is placed in users app, and custom manage.py command comes from this app. In the future we will have more apps and many different objects to generate and I'm not sure if placing factories for them inside users app is a very logical thing.

**Review by ToJestKrzysio — APPROVED** [review]

**Review by MWasile — CHANGES_REQUESTED** [review]
mypy tests failed

File: quiz/backend/users/factories.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.hashers import make_password
```
Comment (SlawCzech) [inline quiz/backend/users/factories.py:2]: This import is not used. Should it stay for generating fake data?

Comment (martynawitkowska) [reply quiz/backend/users/factories.py:2]: Nope just removed it

**Comment by martynawitkowska** [comment]
@MWasile added type hins and mypy passes, but this looks ridiculous 😂 😂😂😂😂😂😂

File: quiz/backend/users/factories.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+from django.utils import timezone
+from factory import Faker, LazyAttribute
+from factory.django import DjangoModelFactory
+from factory.fuzzy import FuzzyDateTime
```
[the remark this answers is not in the document]
Comment (martynawitkowska) [reply quiz/backend/users/factories.py]: changed to Faker class with date_time provider

**Review by SlawCzech — APPROVED** [review]
Looks great, but most importantly - it works :)

**Review by MWasile — APPROVED** [review]
🎸

**Comment by BartekSzymik** [comment]
Super robota. Wszystko działa pięknie

**Pull request merged**